ui/src/components/EntryDetailed/EntrySections.tsx

```diff
@@ -159,24 +165,26 @@ export const EntryBodySection: React.FC<EntryBodySectionProps> = ({
                                                 query={`${selector} == r".*"`}
                                                 updateQuery={updateQuery}
                                             >
-            <table>
-                <tbody>
-                    <EntryViewLine label={'Mime type'} value={contentType} useTooltip={false}/>
-                    {encoding && <EntryViewLine label={'Encoding'} value={encoding} useTooltip={false}/>}
-                </tbody>
-            </table>
-
-            <div style={{display: 'flex', alignItems: 'center', alignContent: 'center', margin: "5px 0"}} onClick={() => setIsWrapped(!isWrapped)}>
+            <div style={{display: 'flex', alignItems: 'center', alignContent: 'center', margin: "5px 0"}}>
                 <div style={{paddingTop: 3}}>
-                    <Checkbox checked={isWrapped} onToggle={() => {}}/>
+                    <Checkbox checked={isPretty} onToggle={() => {setIsPretty(!isPretty)}}/>
+                </div>
+                <span style={{marginLeft: '.2rem'}}>Pretty</span>
+
+                <div style={{paddingTop: 3, paddingLeft: 20}}>
+                    <Checkbox checked={showLineNumbers} onToggle={() => {setShowLineNumbers(!showLineNumbers)}}/>
                 </div>
-                <span style={{marginLeft: '.5rem'}}>Wrap text</span>
+                <span style={{marginLeft: '.2rem'}}>Line numbers</span>
+
+                {isBase64Encoding && <div style={{paddingTop: 3, paddingLeft: 20}}>
+                    <Checkbox checked={decodeBase64} onToggle={() => {setDecodeBase64(!decodeBase64)}}/>
+                </div>}
+                {isBase64Encoding && <span style={{marginLeft: '.2rem'}}>Decode Base64</span>}
             </div>
 
             <SyntaxHighlighter
```

Comment:
Where do you provide the language?

Reply:
@lirazyehezkel the language is detected among [this subset of languages](https://github.com/up9inc/mizu/blob/fix/body-highlight-prettify/ui/src/components/UI/SyntaxHighlighter/index.tsx#L16-L24).